t/plugin/ua-restriction.t

```diff
@@ -340,136 +340,7 @@ User-Agent:foo/bar
 hello world
 
 
-
-=== TEST 18: bypass_missing test, using default, reset conf(part1)
```

Comment:
The `bypass_missing` should work when allowlist or denylist exists and no user agent, the test case is needed

Reply:
Updated.